### ISSUE TITLE
Longer response times

### ISSUE THREAD

**Issue by ThommoMC** [fforchino/vectorx#4]
After installing VectorX I have Vectors response times to Hey Vector are very long, sometimes timing out. I believe the long response times are cause by Initializing the Vector SDK. If the SDK was only Initialized if it was a custom command it would improve response times for non VectorX commands.

**Comment by fforchino** [comment]
Hi, thanks for your point.
I have reworked the startup. I have also added in the setup.sh the
compilation of the go code, this really speed up everything:
Response times to "Hey Vector, what time is it?":
- Regular VectorX code: approx. 10 seconds
- Fixed VectorX code (without sdk init before parsing intent): approx. 8
seconds
- Fixed VectorX code, compiled into binary: approx 6 seconds

I will release the code later today making a new release, look for
RELEASE_10. You will need to relaunch setup.sh to compile the code.

Filippo

Il Sab 4 Feb 2023, 00:14 ThommoMC ***@***.***> ha scritto:
